scheds/rust/scx_rustland/src/main.rs

```diff
@@ -406,8 +406,14 @@ impl<'a> Scheduler<'a> {
                     //
                     // Use the previously used CPU if idle, that is always the best choice (to
                     // mitigate migration overhead), otherwise pick the next idle CPU available.
-                    if !idle_cpus.contains(&task.cpu) {
-                        task.cpu =  *cpu;
+                    if let Some(id) = idle_cpus.iter().position(|&x| x == task.cpu) {
```

Comment:
Bike-shedding but I find it a little confusing to receive position as id. Maybe something like `idx`, `i` or `pos` is better?

Reply:
@htejun yeah it is confusing, id sounds more like the value in this context, not the position, pos is a lot more clear.